packages/core-flows/src/inventory/steps/validate-inventory-items-for-create.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ContainerRegistrationKeys,
+  MedusaError,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+import { InventoryNext } from "@medusajs/types"
+
+export const validateInventoryItemsForCreateStepId =
+  "validate-inventory-items-for-create-step"
+export const validateInventoryItemsForCreate = createStep(
```

Comment:
thought: This step seems to specifically validate if product variant inventory items already exist and I think the name should reflect that.

question: Where is this step used?

todo: The input `inventoryItem` is not used, so we can reduce the type to variants IDs

Reply:
it's not used anywhere, I kept it because of Sebs comment here: https://github.com/medusajs/medusa/pull/6693#discussion_r1525308767